### ISSUE TITLE
Default code style for Erlang should be tabs, tabsize=4, indent=4

### ISSUE THREAD

**Issue by brigadier** [ignatov/intellij-erlang#530]
At the moment code style for erlang code is spaces,2,2 - this is default style for Python code but de-facto standard of Erlang code is tabs, tabsize=4, indent=4

This happens in the PyCharm editor, not sure about other ones.

**Comment by saleyn** [comment]
Is there a way to set the tabsize / indent on the project level so that it applies to the current project only?

**Comment by brigadier** [comment]
Yes you can. Editor -> Code style -> Erlang, In the top left corner of the tab select Scheme: Project and set tabsize/ident.
